project/cmake/CMakeLists.txt

```diff
@@ -450,6 +450,14 @@ configure_file(${PROJECT_SOURCE_DIR}/scripts/common/AddonHelpers.cmake
 configure_file(${PROJECT_SOURCE_DIR}/scripts/common/AddOptions.cmake
                ${CORE_BUILD_DIR}/lib/${APP_NAME_LC}/AddOptions.cmake COPYONLY)
 
+# Documentation
+find_package(Doxygen REQUIRED)
+add_custom_target(doc
+                  COMMAND ${DOXYGEN_EXECUTABLE} ${CORE_SOURCE_DIR}/doxygen_resources/Doxyfile.doxy
+                  COMMAND ${CMAKE_COMMAND} -E echo "Documentation built to: file://${CORE_SOURCE_DIR}/docs/html/index.html"
```

Comment:
Drop the `file://`?
`Documentation built to: file:///home/hudo/dev/kodiws/kodi/docs/html/index.html` doesn't make any sense.